src/charm.py

```diff
@@ -83,9 +83,14 @@ def __init__(self, *args):
         # self._profiling = ProfilingEndpointProvider(
         #     self, jobs=[{"static_configs": [{"targets": ["*:4080"]}]}]
         # )
+        # TODO:
+        #  ingress route provisioning a separate TCP ingress for each receiver if GRPC doesn't work directly
 
-        self._tracing = TracingEndpointProvider(self, host=tempo.host)
-        self._ingress = IngressPerAppRequirer(self, port=self.tempo.tempo_port)
+        self._ingress = IngressPerAppRequirer(self, port=8080, strip_prefix=True)
```

Comment:
Port needs to be tempo http port again. Without nginx we don't publish anything on 8080.